src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -22,7 +23,9 @@ const USER_SETTINGS_KEY = 'selection';
 @observer
 class Selection extends React.Component<Props> {
```

Comment:
Would definitely be in for splitting it up, but the question is where the other components are located. Creating a new component in the `containers` folder feels weird, because that's probably the only place it's going to be used. Having more files here in the `fields` folder is also weird, because all these JS files are usually registered as field types. Having another `Selection` subfolder in this directory might work.

However, I am not sure if I would do that now, because I don't want to rush this decision nor to further delay the merge of #5035.

Reply:
no, i would not do that now. just something to keep in mind for future changes. 

you are right, tha location isnt an easy quesion. to me, a `Selection` folder in the `fields` folder sounds like the best solution at the moment 👍